lib/data/managers/firestore_manager.dart

```diff
@@ -51,6 +52,45 @@ class FireStoreManager {
     return cropsEntitiesWithImagePath;
   }
 
+
+  Future<List<ArticleEntity>> getArticles() async {
+    List<ArticleEntity> articlesEntities;
+
+    // Filters defined by product definition. - ARTICLES
+    var query = Firestore.instance
```

Comment:
Maybe we can do a method that returns this query, with schema as parameter.
Like prepareFirestoreQuery(schema.ARTICLE)
and prepareFirestoreQuery(schema.CROPS)
so it can be reused :)

Reply:
In addition, there are other filters such as the environment and locale that will need to be considered.

Reply:
Done! Thnks for the help